pypeman/remoteadmin.py

```diff
@@ -251,17 +289,20 @@ def stop(self, channel):
         """
         return self.send_command('stop_channel', [channel])
 
-    def list_msg(self, channel, start=0, count=10, order_by='timestamp'):
+    def list_msg(self, channel, start=0, count=10, order_by='timestamp', start_dt=None, end_dt=None):
```

Comment:
can close:
we should have called it `list_msgs` and not `list_msg`

Reply:
done